tests/test_typeform_all_fields.py

```diff
@@ -17,7 +17,7 @@ class TypeformAllFieldsTest(TypeformBaseTest):
     """Ensure running the tap with all streams and fields selected results in the replication of all fields."""
 
     def name(self):
-        return "tap_tester_typeform_all_fields_test"
+        return "tap_tester_typeform_using_shared_token_chaining"
```

Comment:
Names of tests have to be unique and should not change because of this PR.  This should also be changed in the following tests.

Reply:
As per my understanding, when we do token chaining we keep the test names same for all tests so that all tests create connection with same name.

Reply:
This will cause us to not be able to run these tests in parallel. I don't think this is an acceptable solution.  Can you point to where this pattern is in other refresh token repos?

Reply:
Helpscout:
https://github.com/singer-io/tap-helpscout/blob/master/tests/test_helpscout_all_fields.py#L7
https://github.com/singer-io/tap-helpscout/blob/master/tests/test_helpscout_automatic_fields.py#L10

Quickbooks:
https://github.com/singer-io/tap-quickbooks/blob/master/tests/base.py#L42

Reply:
Is there any alternative implementation you can suggest here?

Reply:
I'm not sure what the reason is for the shared name. Can we meet up to discuss why that is required so we can figure out if there is an alternative?

Reply:
Talked IRL.  Decided that if we want to parallelize in the future we can keep the names the same with this solution as  long as we reauth each connection independently.  We will take that solution when parallelization is warranted.  If tests take longer than 15 minutes in series or if they are already running in parallel.  It wasn't clear how this will affect debugging failing test cases.